openapi/build.sbt

```diff
@@ -1,61 +1,52 @@
 import EndpointsSettings._
 import LocalCrossProject._
 
-val testsuite = LocalProject("testsuite")
+lazy val `json-schemaJS` = LocalProject("json-schemaJS")
+lazy val `json-schemaJVM` = LocalProject("json-schemaJVM")
 
-val `json-schema` =
-  crossProject.crossType(CrossType.Pure).in(file("json-schema"))
-    .settings(publishSettings ++ `scala 2.10 to 2.12`: _*)
-    .settings(
-      name := "endpoints-openapi-json-schema",
-      addScalaTestCrossDependency
-    )
-
-val `json-schema-js` = `json-schema`.js
-val `json-schema-jvm` = `json-schema`.jvm
-
-val `json-schema-generic` =
+lazy val `json-schema-generic` =
```

Comment:
I think this should be moved to the `algebra` source tree.

Reply:
Thats a good point. I think they should but on the other hand the openapi falvour of json schema is not standard and thats why I left it here. Cant say if someone wont be confused by the differences if we dont explicitly connect this implementation with `openapi`. What do you think?

Reply:
OK then.